src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+
+    private static final SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
+    }
```

Comment:
Remove the public default constructor from HibernateUtil. The class provides a utility function and is not meant to be instantiated. Adding a private default constructor prevents the creation of HibernateUtil objects.